test/DynamoCoreTests/Graph/Nodes/NodeModelTests.cs

```diff
@@ -49,5 +49,46 @@ public void DictionaryLinkTest()
 
             Assert.AreEqual("http://dictionary.dynamobim.com/2/", dictLink);
         }
+
+        [Test]
+        [Category("UnitTests")]
+        public void SelectNeighborsTest()
+        {
+            string listTestFolder  = Path.Combine(TestDirectory, "core");
+            string testFilePath = Path.Combine(listTestFolder, "Angle.dyn");
+            
+            RunModel(testFilePath);
+
+            NodeModel node = CurrentDynamoModel.CurrentWorkspace.NodeFromWorkspace("dcd9c6c6-6350-4292-a553-c57a764504b4");
+            
+            var countBefore = DynamoSelection.Instance.Selection.Count;
+            Assert.AreEqual(0, countBefore);
+
+            node.SelectNeighbors();
+
+            var countAfter = DynamoSelection.Instance.Selection.Count;
+            Assert.AreEqual(2, countAfter);
+        }
+
+        [Test]
+        [Category("UnitTests")]
+        public void PrintExpressionTest()
+        {
+            string listTestFolder = Path.Combine(TestDirectory, "core");
+            string angleFilePath = Path.Combine(listTestFolder, "Angle.dyn");
+
+            RunModel(angleFilePath);
+
+            NodeModel degreesToRadiansNode = CurrentDynamoModel.CurrentWorkspace.NodeFromWorkspace("dcd9c6c6-6350-4292-a553-c57a764504b4");
+            Assert.AreEqual("(Math.DegreesToRadians Number)", degreesToRadiansNode.PrintExpression());
```

Comment:
hmm, I think this is another good candidate for obsoleting. S expressions were from the old scheme based interpreter for dynamo...

Reply:
@mjkkirschner I see there are a couple instances of usage in Dynamo code, also it is currently marked as public virtual, so should be part of API. Would you like to start obsoletion in Dynamo v2.8?

Reply:
I agree, this should be obsoleted. No tests necessary.

Reply:
There are no real usages of this method, just recursive calls and definitions.